jobs/async-upload/Dockerfile

```diff
@@ -12,38 +14,37 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
     PIP_NO_CACHE_DIR=off \
     PIP_DISABLE_PIP_VERSION_CHECK=on
 
-# Create an unprivileged user
-RUN addgroup --system --gid 1000 app \
-    && adduser  --system --uid 1000 --ingroup app --home /app app
-
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-    curl build-essential libffi-dev libssl-dev skopeo \
```

Comment:
we will need skopeo for the container

Reply:
thanks for spotting!
https://github.com/kubeflow/model-registry/pull/1302/commits/7e3bc28bc86c49780bb5f7c246872931f85ee846